scilpy/io/utils.py

```diff
@@ -376,86 +374,235 @@ def parse_sh_basis_arg(args):
+    """
+    add_volume_screenshot_args(parser, "labelmap", False, "labelmap",
+                               default_cmap=default_cmap,
+                               default_alpha=default_alpha,
+                               cmap_parsing_group=cmap_parsing_group,
+                               opacity_parsing_group=opacity_parsing_group)
+
+
+def add_peaks_screenshot_args(parser, default_width=3.0, default_alpha=1.0,
+                              rendering_parsing_group=None):
+    """
+    Add default arguments for peaks screenshotting.
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    default_width: float, optional
+        Default peak linewidth value.
+    default_alpha: float, optional
+        Default opacity value.
+    rendering_parsing_group: argparse.ArgumentParser object, optional
+        Group to add rendering arguments to, defaults to parser.
+    """
+    parser.add_argument("--peaks", nargs="+",
+                        help="Peaks Nifti image (.nii/.nii.gz).")
+
+    # TODO: check if possible, it would be cool to color by angular lut
+    # or peak intensity. Maybe we do need peeks scalars here, or
+    # its simple to use
+    # parser.add_argument(
+    #    "--peaks_cmap_name", default="viridis",
+    #    help="Colormap name for the peaks image data. [%(default)s]")
+    # parser.add_argument(
+    #    "--peaks_scalars", help="Peaks scalars Nifti image (.nii/.nii.gz) "
+    #                            "used as values for the LUT.")
+
+    rpg = rendering_parsing_group or parser
+    rpg.add_argument("--peaks_width", default=default_width, type=float,
+                     help="Width of the peaks lines. [%(default)s]")
+    rpg.add_argument("--peaks_opacity", type=ranged_type(float, 0., 1.),
+                     default=default_alpha,
+                     help="Opacity value for the peaks overlay. [%(default)s]")
+
+
+def add_overlays_screenshot_args(parser, default_alpha=0.5,
+                                 rendering_parsing_group=None):
+    """
+    Add default arguments for overlays screenshotting.
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    default_alpha: float, optional
+        Default opacity value.
+    rendering_parsing_group: argparse.ArgumentParser object, optional
+        Group to add rendering arguments to, defaults to parser.
+    """
+    parser.add_argument("--overlays", nargs="+",
+                        help="3D Nifti image to overlay (.nii/.nii.gz).")
+
+    rpg = rendering_parsing_group or parser
+    rpg.add_argument("--overlays_as_contours", action='store_true',
+                     help="Display overlays contours and reduce the opacity "
+                          "of their inner region (see the "
+                          "`--overlays_opacity` argument).")
+    rpg.add_argument("--overlays_colors", nargs="+",
+                     type=ranged_type(int, 0, 255),
```

Comment:
Cool! I didn't know this was possible. Will remember this for other scripts, for sure!